### DISCUSSION TITLE
What is wrong with filer upsert query in 3.80?

### DISCUSSION THREAD

**Discussion by gutleib** [seaweedfs/seaweedfs#6293] [Q&A]
Hi!
I'm bringing up Seaweedfs cluster with multiple filers, and decided to use shared Mariadb+Galera as storage for filer db. It's relatively efficient, simple to deploy and uses not a lot of resources. And it's really mature as a DB.
This way filers metadata is in sync'd by Galera.
But I can't figure out how to connect more than 1 filer: it fails with some comments about Maradb syntax. This doesn't seem to be normal...

```shell
filer_server.go:203 10.0.0.3:8888 bootstrap from [node_type:"filer"  address:"10.0.0.2:8889"  is_add:true  created_at_ns:1732510290606762095 node_type:"filer"  address:"10.0.0.3:8888"  is_add:true  created_at_ns:1732634996877733040]: insert /: Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'AS `new` ON DUPLICATE KEY UPDATE `meta` = `new`.`meta`' at line 1
```

**Comment by kmlebedev** [comment]
try use
```
INSERT INTO `%s` (`dirhash`,`name`,`directory`,`meta`) VALUES (?,?,?,?) ON DUPLICATE KEY UPDATE `meta` = VALUES(`meta`)
```
